packages/vuetify/src/components/VProgressLinear/VProgressLinear.tsx

```diff
@@ -18,6 +18,8 @@ export const VProgressLinear = defineComponent({
   name: 'VProgressLinear',
 
   props: {
+    absolute: Boolean,
+    bottom: Boolean,
```

Comment:
could support `top` as well

Reply:
add the top property and update documentation